tvb_storage/tvb/encryption/data_encryption_handler.py

```diff
@@ -176,33 +182,31 @@ def sync_folders(folder):
 
         project_name = os.path.basename(folder)
         encrypted_folder = DataEncryptionHandler.compute_encrypted_folder_path(folder)
-        try:
-            project = dao.get_project_by_name(project_name)
-        except NoResultFound:
+
+        if os.path.exists(encrypted_folder) and os.path.joins(folder):
```

Comment:
here I think the correct condition is if os.path.exists(encrypted_folder) or os.path.exists(folder)

Reply:
Yeah, I misstyped, thanks for noticing. Is the rest okay here?